backend/open_webui/utils/access_control.py

```
@@ -1,5 +1,5 @@
 from typing import Optional, Union, List, Dict, Any
```

Ruff (0.8.2)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)